_clients/data-prepper/migrate-open-distro.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: Migrating from Open Distro Data Prepper
+parent: Data Prepper
+nav_order: 2
+---
+
+# Migrating from Open Distro Data Prepper
+
+Beginning with Data Prepper 1.1, there is only one distribution of Data Prepper: OpenSearch Data Prepper. This document helps existing users migrate from the old Open Distro Data Prepper to OpenSearch Data Prepper.
```

Comment:
Is "the old" necessary here? I think our meaning is probably sufficiently conveyed without it.